api/src/main/java/org/pmiops/workbench/actionaudit/bucket/BucketAuditEntry.java

```diff
@@ -0,0 +1,69 @@
+package org.pmiops.workbench.actionaudit.bucket;
+
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class BucketAuditEntry {
+
+  private String petAccount;
+  private long fileLengths;
+  private String googleProjectId;
+  private String bucketName;
+  private OffsetDateTime minTime;
+  private OffsetDateTime maxTime;
+
+  public String getPetAccount() {
+    return petAccount;
+  }
+
+  public void setPetAccount(String petAccount) {
+    this.petAccount = petAccount;
+  }
+
+  public long getFileLengths() {
+    return fileLengths;
+  }
+
+  public void setFileLengths(long fileLengths) {
+    this.fileLengths = fileLengths;
+  }
+
+  public OffsetDateTime getMinTime() {
```

Comment:
Prefer to use Instant over timezone (or others that use the default system time zone). 
More in general, using java.time is preferred. 
[go/javatime](https://goto.google.com/javatime)